docs_src/security/Ch-SecretStore.md

```diff
@@ -1134,6 +290,5 @@ man-style documentation:
 
 -   [security-file-token-provider](./security-file-token-provider.1.md) -
     Generate Vault tokens for EdgeX services
--   [security-secrets-setup](./security-secrets-setup.1.md) - Creates an
-    on-device public-key infrastructure (PKI) to secure microservice
-    secret management
+-   [secrets-config](./secrets-config.1.md) - Utility for secrets management.
+-   [secrets-config-proxy](./secrets-config-proxy.1.md) - "proxy" subcommand for managing proxy secrets.
```

Comment:
Ok, here is where it is referenced. Is there are reason we don't have it in the main doc menu, i.e. in the mkdocs.yml?

Reply:
Sure, I will add it.